osmaxx/core/templates/pages/about_us.html

```diff
@@ -8,6 +8,12 @@
         OSMaxx (short for "OpenStreetMap arbitrary excerpt export") cuts out OpenStreetMap data, processes it to
         geodata, converts it to typical GIS fileformats and offers those for download.
     </p>
+    <h3>Documentation</h3>
+    <h4>Schema</h4>
+    <p>
+        You can find the online documentation of the schema on GitHub:
+        <a href="https://github.com/geometalab/osmaxx-docs/blob/master/documentation.md">Export Schema Documentation</a>.
```

Comment:
Oops, should have removed that one back [when I moved the documentation](https://github.com/geometalab/osmaxx/commit/ee63b85d0f8d5c635749e07a65e1e5d324e6138b) to [this repo](https://github.com/geometalab/osmaxx/blob/develop/docs/osmaxx_data_schema.md). Please link to the new location.